app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ReviewsController < ApplicationController
+  # This controller is nested - its routes are under /restaurants/:restaurant_id/...
+
+  before_action :query_review, only: [:show, :update, :destroy]
```

Comment:
A shorter version for this would be:
```ruby
only: %i[show update destroy]
```

This also works with `(`,`)` or `{`,`}`. I personally prefer regular parentheses.